src/components/editor/EditorAppController.jsx

```diff
@@ -137,7 +144,7 @@ export default class EditorAppController extends BaseComponent {
             title={"Editor Tools"}
             iconElementRight={this.isLoggedIn() ?
```

Comment:
I didn't modify this part, but will change it now!

Reply:
Okay fair enough, it would work if `this.isLoggedIn` returned `undefined` or `null` in the falsy instance I guess, but yep, just keep it like that then!